src/google-classroom-extractor/tests/api/courses/courses-initial.csv

```diff
@@ -0,0 +1,9 @@
+id,name,section,descriptionHeading,description,room,ownerId,creationTime,updateTime,enrollmentCode,courseState,alternateLink,teacherGroupEmail,courseGroupEmail,guardiansEnabled,calendarId,SyncNeeded,CreateDate,LastModifiedDate
```

Comment:
initial test has regular table with one course with update, two courses that are in both but unchanged, and one course not in the sync table